source/includes/changestream-paradigm-warning.rst

```diff
@@ -1,4 +1,8 @@
 .. warning::
 
-   Attempting to use a ``ChangeStream`` as an ``Iterator`` that is in ``EventEmitter`` mode will cause an error,
-   and attempting to use a ``ChangeStream`` as an ``EventEmitter`` in ``Iterator`` mode will cause an error.
+   Using a ``ChangeStream`` in ``EventEmitter`` and ``Iterator`` mode
+   concurrently is not supported and causes an error. This is to prevent
+   undefined behavior, where it cannot be guaranteed which consumer will
```

Comment:
A couple items here:
1) Similar to the prior comment, if you specify "the driver" here, you can avoid passive voice. E.g. "...the driver cannot guarantee..."
2) "will receive" is future tense. You can convert this into present with minor rewording, e.g. "which consumer receives documents first".

Reply:
Thanks for the suggestions!